docs/source/guide/lre-1-intro.md

```diff
@@ -0,0 +1,152 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# How do I use LRE?
+
+LRE works in two main steps: generate noise-scaled circuits, and apply inference to results from executed circuits.
```

Comment:
```suggestion
LRE works in two main steps: generate layered noise-scaled circuits in a specific pattern, and apply inference to results from executed circuits.
```

Reply:
I added a few more details in https://github.com/unitaryfund/mitiq/pull/2535/commits/f2f38290f13995f61aa2b555ea47ad94f3845eac.